docs/volumegroupreplication.md

```diff
@@ -0,0 +1,58 @@
+# VolumeGroupReplication
+
+VolumeGroupReplication is a namespaced resource that contains references to storage object to be grouped and replicated, VolumeGroupReplicationClass corresponding to the driver providing replication, VolumeGroupContent and VolumeReplication CRs.
```

Comment:
`VolumeGroupContent` should be `VolumeGroupReplicationContent`?

Reply:
Done